iam-login-service/src/test/java/it/infn/mw/iam/test/api/account/password/PasswordUpdateTests.java

```diff
@@ -211,30 +205,68 @@ public void testUpdatePasswordTooShortPasswordAccess() {
     String currentPassword = "password";
     String newPassword = "pass";
     String accessToken = passwordTokenGetter().port(iamPort)
-      .username(testUser.getUserName())
-      .password(currentPassword)
-      .getAccessToken();
+        .username(testUser.getUserName())
+        .password(currentPassword)
+        .getAccessToken();
+
+    doPost(accessToken, currentPassword, newPassword).statusCode(HttpStatus.BAD_REQUEST.value());
+  }
+
+  @Test
+  public void testUpdatePasswordWithWeakPasswordAccess() {
+
+    String currentPassword = "password";
+    String newPassword = "newweakpassword";
+    String accessToken = passwordTokenGetter().port(iamPort)
+        .username(testUser.getUserName())
+        .password(currentPassword)
+        .getAccessToken();
+
+    doPost(accessToken, currentPassword, newPassword).statusCode(HttpStatus.BAD_REQUEST.value());
+  }
+
+  @Test
+  public void testUpdatePasswordWithWeakPasswordWithoutSpecialCharts() {
```

Comment:
just a typo in the name of the test: testUpdatePasswordWithWeakPasswordWithoutSpecial**Chars**